src/Microsoft.AspNetCore.Razor.Tools/project.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "Microsoft.AspNetCore.Razor.Design": "1.0.0-*",
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview3-003890",
```

Comment:
@pranavkm We need to pull in latest bits to make this happy 😄 . Enables us to also nuke the NuGet.config entry above.